kotlin/open-mobile-map-package/README.md

```diff
@@ -0,0 +1,48 @@
+# Open mobile map package
+
+Display a map from a mobile map package.
+
+![Open mobile map package Sample](open-mobile-map-package.png)
+
+## Use case
+
+An .mmpk file is an archive containing the data (specifically, basemaps and features) used to display an offline map.
+A mobile map package may need to be unpacked to a directory to allow read support for certain data types.
+
+## How to use the sample
+
+When the sample opens, it will automatically display the map in the mobile map package. Pan and zoom to observe the data from the mobile map package.
+
+## How it works
+
+1. Create a `MobileMapPackage` specifying the path to the .mmpk file.
+2. Use `MobileMapPackage.isDirectReadSupportedAsync()` to check whether the package can be read in the archived form (.mmpk) or whether it needs to be unpacked.
```

Comment:
the direct read support can be removed--it's not relevant in our API anymore (but was in 100.5!)

Reply:
sorry about that!